ui/clubhouse.js

```diff
@@ -1319,13 +1321,40 @@ function enable() {
         if (actor && actor.metaWindow)
             destroyNotifyWindow(actor.metaWindow);
     });
+
+    // Check the onboarding extension to show the notification over the highlighting
+    ONBOARDING_PROXY = new Gio.DBusProxy.new_for_bus_sync(
+        Gio.BusType.SESSION,
+        0, null,
+        'com.endlessm.onboarding',
+        '/com/endlessm/onboarding',
+        'com.endlessm.onboarding',
+        null);
```

Comment:
If the DBus path / interface is not there, this doesn't fail, and the code continues working the same. This way this is working even if the onboarding extension is loaded after this proxy is created, because the connected callback will be called with the signal, even if the extension is not loaded when this code is executed.

I've tested this case because I don't know if the hack extension will be loaded before the onboarding service is created, but it looks like the connect is just listening the dbus event and no matter if this interface exists or not.